components/org.wso2.carbon.identity.application.authenticator.biometric.device.handler/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/device/handler/model/Device.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.biometric.device.handler.model;
+
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * This class contains the attributes and operations of a device.
+ */
+public class Device implements Serializable {
+    private String deviceId;
+    private String userId;
+    private String deviceName;
+    private String deviceModel;
+    private String pushId;
+    private String publicKey;
+    private Date registrationTime;
+    private Date lastUsedTime;
+
+    public Device() {
+    }
+
+
+    public Device(String deviceId, String userId, String deviceName, String deviceModel,
+                  String pushId, String publicKey, Date registrationTime, Date lastUsedTime) {
+        this.deviceId = deviceId;
```

Comment:
```suggestion
                  String pushId, String publicKey, Date registrationTime, Date lastUsedTime) {
                  
        this.deviceId = deviceId;
```
Kindly check formatting in other places.